app/static/js/createEvents.js

```diff
@@ -484,3 +484,88 @@ $(".startDatePicker, .endDatePicker").change(function () {
 
   setCharacterLimit($("#inputCharacters"), "#remainingCharacters"); 
 });
+
+function inviteCohorts(eventId, selectedCohorts) {
+  return $.ajax({
+    type: "POST",
+    url: `/event/${eventId}/invite-cohorts`,
+    contentType: "application/json",
+    data: JSON.stringify({
+      cohorts: selectedCohorts
+    }),
+    success: function(response) {
+      if (response.success) {
+        msgFlash("Cohorts successfully invited!", "success");
+        updateCohortStatus(eventId);
+      } else {
+        msgFlash("Error inviting cohorts", "danger");
+      }
+    }
+  });
+}
+
+function getCohortStatus(eventId) {
+  return $.ajax({
+    type: "GET",
+    url: `/event/${eventId}/cohort-status`,
+    success: function(response) {
+      displayCohortStatus(response.invited_cohorts);
+    }
+  });
+}
+
+function displayCohortStatus(invitedCohorts) {
+  $(".cohort-status").remove();
+  
+  invitedCohorts.forEach(cohort => {
+    const invitedDate = new Date(cohort.invited_at).toLocaleDateString();
+    const cohortElement = $(`#cohort-${cohort.year}`);
+    
+    if (cohortElement.length) {
+      const statusHtml = `
+        <div class="cohort-status mt-1 ml-4">
+          <small class="text-success">
+            <i class="fas fa-check-circle"></i>
+            Invited on ${invitedDate}
+          </small>
+        </div>
+      `;
+      cohortElement.closest('.form-check').append(statusHtml);
+      cohortElement.prop('disabled', true);
+      cohortElement.closest('.form-check-label').addClass('text-muted');
+    }
+  });
+}
+
+function updateCohortStatus(eventId) {
```

Comment:
Unless we are planning on doing some extra logic in this function we should probably just call getCohortStatus instead of defining this new function just to call getCohortStatus.